autodispose-android-archcomponents/src/main/java/com/uber/autodispose/android/lifecycle/AndroidLifecycleScopeProvider.java

```diff
@@ -54,41 +54,84 @@
         }
       };
 
+  private final Function<Lifecycle.Event, Lifecycle.Event> correspondingEvents;
+
   /**
    * Creates a {@link AndroidLifecycleScopeProvider} for Android LifecycleOwners.
    *
-   * @param owner the owner to scope for
+   * @param owner the owner to scope for.
    * @return a {@link AndroidLifecycleScopeProvider} against this owner.
    */
   public static AndroidLifecycleScopeProvider from(LifecycleOwner owner) {
     return from(owner.getLifecycle());
   }
 
   /**
+   * Creates a {@link AndroidLifecycleScopeProvider} for Android LifecycleOwners.
+   *
+   * @param owner the owner to scope for.
+   * @param untilEvent the event until the scope is valid.
+   * @return a {@link AndroidLifecycleScopeProvider} against this owner.
+   */
+  public static AndroidLifecycleScopeProvider from(LifecycleOwner owner,
```

Comment:
nit: put both of these on the next line or at least make `until` event use a normal trailing indent

Reply:
Fixed!